scripts/d-i/xenial/preseed_video.cfg

```diff
@@ -29,7 +29,7 @@ d-i preseed/early_command string cd /tmp && wget http://$url/ec/early_command.sh
 # d-i preseed/run string /ec/early_command.sh
 
 ### cfk - get late script from shaz, run it.
-d-i preseed/late_command string cd /target/tmp && unset http_proxy; wget http://$url/late_command.sh && chmod u+x late_command.sh && ANSIBLE_UNDER_DI=1 in-target /tmp/late_command.sh $url $(debconf-get mirror/suite) $(debconf-get passwd/username)
+d-i preseed/late_command string cd /target/tmp && unset http_proxy; wget http://$url/d-i/late_command.sh && chmod u+x late_command.sh && ANSIBLE_UNDER_DI=1 in-target /tmp/late_command.sh $url $(debconf-get mirror/suite) $(debconf-get passwd/username) $(debconf-get lc/playbook_repo) $(debconf-get lc/playbook_branch) $(debconf-get lc/inventory_repo) $(debconf-get lc/inventory_branch)
```

Comment:
my attempt at splitting up the pressed file into chunks that need swapping now and then. 

preseed.cfg just links to 3 or 4 more files.  If I remove the -disk one, then the installer will prompt me for disk parameters. 

Not sure how useful it really is.